src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx

```diff
@@ -48,6 +78,156 @@ describe('JiraIssueList', () => {
     expect(screen.getByText('ISSUE_2')).toHaveAttribute('href', 'https://jira.com/ISSUE_2')
   })
 
+  test('it should display "Add Jira Issue Modal" on button click', async () => {
+    renderComponent()
+
+    const jiraModalButton = screen.getByRole('button', { name: 'cf.featureFlags.jira.newJiraIssueButton' })
+    expect(jiraModalButton).toBeInTheDocument()
+    userEvent.click(jiraModalButton)
+
+    //assert dialog elements appear
+    expect(screen.getByText('cf.featureFlags.jira.jiraModalTitle')).toBeInTheDocument()
+    expect(screen.getByPlaceholderText('- cf.featureFlags.jira.inputPlaceholder -')).toBeInTheDocument()
+    expect(screen.getByText('add')).toBeInTheDocument()
+    expect(screen.getByText('cancel')).toBeInTheDocument()
+  })
+
+  test('it should let the user save a Jira Issue correctly', async () => {
+    renderComponent()
+
+    const jiraModalButton = screen.getByRole('button', { name: 'cf.featureFlags.jira.newJiraIssueButton' })
+    expect(jiraModalButton).toBeInTheDocument()
+    userEvent.click(jiraModalButton)
+
+    // type into issue search field
+    expect(screen.getByPlaceholderText('- cf.featureFlags.jira.inputPlaceholder -')).toBeInTheDocument()
+    userEvent.click(screen.getByPlaceholderText('- cf.featureFlags.jira.inputPlaceholder -'))
+    await userEvent.type(screen.getByPlaceholderText('- cf.featureFlags.jira.inputPlaceholder -'), 'FD-7', {
+      allAtOnce: true
+    })
+
+    // select issue from dropdown
+    await waitFor(() => expect(screen.getAllByRole('listitem')).toHaveLength(1))
+    expect(screen.getByRole('listitem')).toHaveTextContent('FD-7')
+    userEvent.click(screen.getByRole('listitem'))
+    await waitFor(() =>
+      expect(screen.getByPlaceholderText('- cf.featureFlags.jira.inputPlaceholder -')).toHaveValue('FD-7')
+    )
+
+    // submit form
+    userEvent.click(screen.getByText('add'))
```

Comment:
Best to be super specific that we're wanting to click a button with that text.
```suggestion
    userEvent.click(screen.getByRole('button', { name: 'add' }))
```